entity-framework/core/what-is-new/ef-core-5.0/plan.md

```diff
@@ -0,0 +1,227 @@
+
+* Work on Linux, Mac, and Windows
+* Be a good experience on the command line
+* Support scenarios with containers
+* Work with commonly used real-world deployment tools/flows
+* Integrate into at least Visual Studio
+
+The result is likely to be many small improvements in EF Core (for example, better Migrations on SQLite), together with guidance and longer-term collaborations with other teams to improve end-to-end experiences that go beyond just EF.
+
+## EF Core platforms experience 
+
+Lead developers: @roji and @bricelam
+
+Tracked by [#19588](https://github.com/dotnet/efcore/issues/19588)
+
+T-shirt size: L
+
+We have good guidance for using EF Core in traditional MVC-like web applications. Guidance for other platforms and application models is either missing or out-of-date. For EF Core 5.0 we plan to investigate, improve, and document the experience of using EF Core with:
+
+* Blazor
+* Xamarin, including using the AOT/linker story
+* WinForms/WPF/WinUI and possibly other U.I. frameworks
+
+This is likely to be many small improvements in EF Core, together with guidance and longer-term collaborations with other teams to improve end-to-end experiences that go beyond just EF.
+
+Specific areas we plan to look at are:
+
+* Deployment, including the experience for using EF tooling such as for Migrations
+* Application models, including Xamarin and Blazor, and probably others
+* SQLite experiences, including the spatial experience and table rebuilds
+* AOT and linking experiences
+* Diagnostics integration, including perf counters
+
+## Performance
+
+Lead developer: @roji
+
+Tracked by [issues labeled with `area-perf` in the 5.0 milestone](https://github.com/dotnet/efcore/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3Aarea-perf+milestone%3A5.0.0+)
+
+T-shirt size: L
+
+For EF Core we plan to improve our suite of performance benchmarks and make directed performance improvements to the runtime. In addition, we plan to complete the new ADO.NET batching API which was prototyped during the 3.0 release cycle. Also at the ADO.NET layer, we plan additional performance improvements to the Npgsql provider.
+
+As part of this work we also plan to add ADO.NET/EF Core performance counters and other diagnostics as appropriate.
+
+## Architectural/contributor documentation
+
+Lead documenter: @ajcvickers
+
+Tracked by [#1920](https://github.com/aspnet/EntityFramework.Docs/issues/1920)
+
+T-shirt size: L
+
+The idea here is to make it easier to understand what is going on in the internals of EF Core. This can be useful to anyone using EF Core, but the primary motivation is to make it easier for external people to:
+
+* Contribute to the EF Core code
+* Create database providers
+* Build other extensions
+
+## Microsoft.Data.Sqlite documentation
+
+Lead documenter: @bricelam
+
+Tracked by [#1675](https://github.com/aspnet/EntityFramework.Docs/issues/1675)
```

Comment:
Status: Done ([read the docs](http://docs.microsoft.com/dotnet/standard/data/sqlite/))